lib/diagnostic.js

```diff
@@ -15,11 +15,26 @@
 
 const fs = require('fs'); // jshint ignore: line
 
+
 const sumLines = (filename, callback) => {
-  /* your response here */
-  callback();
-};
 
+  fs.readFile(filename, { encoding: 'utf8' }, callback(error, data) => {
+    let pojo, json;
+    if (error) {
+      console.error(error.stack);
+      return;
+    }
+    try {
+      pojo = JSON.parse(data);
+    } catch (error) {
+      console.error(error.stack);
+      return;
+    }
+
+    console.log(pojo);
+  });
+};
+sumLines('../data/integers.txt', callback(null, sum));
```

Comment:
I think you are confusing some things. 
1) The purpose of this code is to read a .txt file line by line, and add those number together. There is no JSON involved. 
2) on line 21, you have `callback(error, data) => { ... }` as your third argument. What is that syntax actually doing? It's calling a function called `callback` with the arguments `error` and `data`. 
3) between lines 21 and 34, are you actually calculating the sum??
4) line 37 is unnecessary. You do not need to call your own function. It's the job of the test (and your instructors) to call the function.